Source/WebKit/GPUProcess/mac/GPUProcessMac.mm

```diff
@@ -63,8 +63,19 @@
 void GPUProcess::initializeProcessName(const AuxiliaryProcessInitializationParameters& parameters)
 {
 #if !PLATFORM(MACCATALYST)
+    // Register the application. This will also check in with Launch Services.
+    // This is necessary to set the process name.
+    _RegisterApplication(nullptr, nullptr);
```

Comment:
Would it be sufficient to just check in with Launch Services? I do not think the Networking process calls _RegisterApplication?